packages/components/src/HeaderBar/__snapshots__/HeaderBar.snapshot.test.js.snap

```diff
@@ -1,5803 +1,10184 @@
-        />
-      </svg>
-    </symbol>
-    <symbol
-      id="talend-question-circle"
-    >
-      <svg
-        viewBox="0 0 16 16"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M13.63934,2.36633A7.68945,7.68945,0,0,0,7.98126.00024,7.68945,7.68945,0,0,0,2.32318,2.36633,7.54891,7.54891,0,0,0,0,8a7.6905,7.6905,0,0,0,2.32318,5.6712,7.80088,7.80088,0,0,0,5.65808,2.32855,7.80088,7.80088,0,0,0,5.65808-2.32855A7.72494,7.72494,0,0,0,16,8,7.58242,7.58242,0,0,0,13.63934,2.36633ZM9,14H7V12H9Zm2.1084-6.75024a3.72654,3.72654,0,0,1-.46014.79755,6.53576,6.53576,0,0,1-.58228.66925,7.38283,7.38283,0,0,0-.56738.6427,2.9032,2.9032,0,0,0-.41272.72528A1.99785,1.99785,0,0,0,8.97864,11H7.01727a2.59439,2.59439,0,0,1,.03864-1.0791A3.31245,3.31245,0,0,1,7.415,9.06146,3.80161,3.80161,0,0,1,7.94373,8.373c.19946-.2049.38525-.4054.55981-.59851a4.37632,4.37632,0,0,0,.44391-.57928A1.16243,1.16243,0,0,0,9.132,6.56134a1.07994,1.07994,0,0,0-.32294-.8609,1.18933,1.18933,0,0,0-.81177-.28009,1.5239,1.5239,0,0,0-.8429.2359,4.0454,4.0454,0,0,0-.71948.597L5.178,4.89557a4.63289,4.63289,0,0,1,1.303-1.17786A3.16688,3.16688,0,0,1,8.1507,3.26514,4.12294,4.12294,0,0,1,9.38391,3.445a2.72745,2.72745,0,0,1,.99628.5528,2.65956,2.65956,0,0,1,.66711.92432,3.1968,3.1968,0,0,1,.24438,1.29578A2.64809,2.64809,0,0,1,11.1084,7.24976Z"
-        />
-      </svg>
-    </symbol>
-    <symbol
-      id="talend-search"
-    >
-      <svg
-        version="1.1"
-        viewBox="0 0 16 16"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M7.5,14C3.9,14,1,11.1,1,7.5S3.9,1,7.5,1S14,3.9,14,7.5S11.1,14,7.5,14z M7.5,3C5,3,3,5,3,7.5S5,12,7.5,12S12,10,12,7.5S10,3,7.5,3z"
-        />
-        <path
-          d="M13.5,12c-0.4,0.6-0.9,1.1-1.5,1.5l2.5,2.5l1.5-1.5L13.5,12z"
-        />
-      </svg>
-    </symbol>
-    <symbol
-      id="talend-share-alt"
-    >
-      <svg
-        version="1.1"
-        viewBox="0 0 16 16"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M12.5,11c-0.7,0-1.4,0.3-1.8,0.8l-4.8-3C5.9,8.6,6,8.3,6,8S5.9,7.4,5.8,7.2l4.8-3C11.1,4.7,11.8,5,12.5,5C13.9,5,15,3.9,15,2.5C15,1.1,13.9,0,12.5,0S10,1.1,10,2.5c0,0.3,0.1,0.6,0.2,0.8l-4.8,3C4.9,5.8,4.2,5.5,3.5,5.5C2.1,5.5,1,6.6,1,8c0,1.4,1.1,2.5,2.5,2.5c0.7,0,1.4-0.3,1.8-0.8l4.8,3c-0.1,0.3-0.2,0.5-0.2,0.8c0,1.4,1.1,2.5,2.5,2.5s2.5-1.1,2.5-2.5C15,12.1,13.9,11,12.5,11z"
-        />
-      </svg>
-    </symbol>
-    <symbol
-      id="talend-star"
-    >
-      <svg
-        version="1.1"
-        viewBox="0 0 16 16"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <polygon
-          points="8,0 10.5,5 16,5.8 12,9.7 12.9,15.2 8,12.6 3.1,15.2 4,9.7 0,5.8 5.5,5 "
-        />
-      </svg>
-    </symbol>
-    <
```

Comment:
10 000 lignes of snashots is way to much.
either revert to storyshots or review all the tests to make only interesting ones